python/ray/workers/default_worker.py

```diff
@@ -134,6 +134,11 @@
     default=False,
     action="store_true",
     help="True if Ray debugger is made available externally.")
+parser.add_argument(
```

Comment:
Can we make this a private/experimental API for now?

Reply:
I set `hidden=True`, is it enough?